privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -59,8 +59,11 @@ <h1>{% ftlmsg 'multi-part-onboarding-premium-get-domain' %}</h1>
                         {% endif %}
                         
                         <div class="c-premium-onboarding-action-completed {% if user_profile.subdomain %} is-visible {% endif %}">
-                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
-                            <samp class="js-premium-onboarding-domain-registration-preview">{{ user_profile.custom_domain }}</samp>
```

Comment:
I couldn't test it because the removal of `js-premium-onboarding-domain-registration-preview` caused a JS error:

https://user-images.githubusercontent.com/4251/151981857-63bd4653-dcf3-420d-b356-fc981cc8182a.mp4

Reply:
I removed instances of `domainPreview` and any unused parameters.